media/gpu/v4l2/generic_v4l2_device.cc

```diff
@@ -474,7 +474,11 @@ bool GenericV4L2Device::OpenDevicePath(const std::string& path, Type type) {
     return false;
 
 #if BUILDFLAG(USE_LIBV4L2)
+#if BUILDFLAG(USE_LINUX_V4L2)
+  if (
+#else
   if (type == Type::kEncoder &&
+#endif
```

Comment:
Added the lines for USE_LINUX_V4L2 flag.
I hope this would make sense.

Reply:
yes, thanks